migrations/20190115120821_create_transactions_summary_table.up.sql

```diff
@@ -1,9 +1,11 @@
 CREATE TABLE IF NOT EXISTS TransactionSummary(
+    id int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
 	userId int(11) UNSIGNED NOT NULL,
     stockId int(11) UNSIGNED NOT NULL,
     stockQuantity bigint(11) SIGNED NOT NULL,
     price float(11,2) UNSIGNED NOT NULL,
+    PRIMARY KEY (id),
+    UNIQUE KEY NONCLUSTERED (userId, stockId),
 	FOREIGN KEY (userId) REFERENCES Users(id),
-    FOREIGN KEY (stockId) REFERENCES Stocks(id),
-	PRIMARY KEY (userId, stockId)
-	) AUTO_INCREMENT = 1;
+    FOREIGN KEY (stockId) REFERENCES Stocks(id)
+) AUTO_INCREMENT=1;
```

Comment:
Shouldn't a new migration be created rather than changing previous one? Previous PR was merged running migrations won't update the schema